flocker/node/agents/ebs.py

```diff
@@ -107,6 +115,63 @@ def _wait_for_volume(expected_volume,
             )
 
 
+def _check_blockdevice_size(device, size):
+    """
+    Helper function to check if the size of block device with given
+    suffix matches the input size.
+
+    :param unicode device: Name of the block device to check for size.
+    :param int size: Size, in SI metric bytes, of device we are interested in.
+
+    :returns: True if a block device with given name has given size.
+        False otherwise.
+    """
+    device_name = b"/dev/" + device.encode("ascii")
+
+    # Retrieve size of device as OS sees it using `lsblk`.
+    # Requires util-linux-ng package on CentOS, and
+    # util-linux on Ubuntu.
+    # Required package is installed by default
+    # on Ubuntu 14.04 and CentOS 7.
+    command = [b"/bin/lsblk", b"-nb", b"-o", b"SIZE", device_name]
```

Comment:
`--noheader` would be nicer than `-n` here.  Likewise for `--bytes` and `--output`.